app/src/main/res/values/strings_questionnaire.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="questionnaire_title_age">Age</string>
+    <string name="questionnaire_description_age">How old are you?</string>
+    <string-array name="questionnaire_keys_age">
+        <item>10~19</item>
+        <item>20~29</item>
+        <item>30~39</item>
+        <item>40~49</item>
+        <item>50~59</item>
+        <item>60~</item>
+        <item>secret</item>
+    </string-array>
+    <string-array name="questionnaire_values_age">
```

Comment:
**[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: "questionnaire_values_age" is not translated in "ja" (Japanese) [MissingTranslation]